packages/spatie-laravel-media-library-plugin/src/Tables/Columns/SpatieMediaLibraryImageColumn.php

```diff
@@ -57,7 +58,7 @@ public function getImageUrl(?string $state = null): ?string
             $media = $record->getRelationValue('media')->first(fn (Media $media): bool => $media->uuid === $state);
 
             if (! $media) {
-                continue;
+                return $record->getFirstMediaUrl($this->getCollection()); // @phpstan-ignore-line
```

Comment:
This will break using the column on a -many relationship since once one record doesn't have media, the whole thing will fail